chimney/src/main/scala/chimney/package.scala

```diff
@@ -0,0 +1,11 @@
+/**
+  * Aliases for easier access to the Chimney API.
+  * */
+package object chimney {
+  import io.scalaland.{chimney => ch}
+
+  val dsl: ch.dsl.type = ch.dsl
+
+  type Patcher[T, Patch] = ch.Patcher[T, Patch]
+  type Transformer[From, To] = ch.Transformer[From, To]
+}
```

Comment:
I believe we need to alias also companion objects of `Patcher` and `Transformer` types. For instance one could use `Patcher.derive` or `Transformer.derive` in order to run derivation macro. Without these aliases it wouldn't be possible.

Reply:
Right, for some reason I thought they were implicits and nobody would call them directly - not true :P

Reply:
I would mainly expect such usage too but since these symbols are public, we can't assume nobody uses them directly.